rust-toolchain

```diff
@@ -0,0 +1 @@
+1.51.0
```

Comment:
We're currently staying 1 behind latest stable as wasmer crates get used as a libraries and it makes the user experience a bit better. We used to get a lot of error reports and confusion about wasmer only working on nightly when we were using the latest stable. The reason is that the error message you'll get when using rustc version N-1 to compile code using a feature stabilized in version N says something like "feature X is not stable, you can use it on the nightly compiler with #[feature(X)]". This understandably leads people to believe that wasmer only works on nightly.

It's also nice as a library to not rush our users to update too much. In the future we may try to support many Rust versions, but for now 1 behind latest stable is a good compromise.

Reply:
That is good to know how 1 behind latest stable helps! I'll update it to 1.50